spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -2099,15 +2116,26 @@ protected List<DataFile> currentDataFiles(Table table) {
 
   protected List<Object[]> currentData() {
     return rowsToJava(
-        spark.read().format("iceberg").load(tableLocation).sort("c1", "c2", "c3").collectAsList());
+        spark
+            .read()
+            .option(SparkReadOptions.SPLIT_SIZE, 1024 * 1024 * 32)
```

Comment:
nit: Is there a reason some places use 32MB and others 64MB for `SPLIT_SIZE`?  If not, shall we unify them?